d8error.py

```diff
@@ -80,9 +81,39 @@ def furtherTips(self):
     def print(self, i):
         display(Markdown)
     def title(self):
-        display(Markdown("## **Uh-o it seems we have an error!**"))
+        display(Markdown("## **There seems to be a <font color='red'>" + self.errorname+ "<font>**" + "."))
     def default(self):
-        display(Markdown("It seems we have a "+self.errorname+ ". " +self.errorname+ "s are usually because of:"))
+        display(Markdown("Here is some possible reasons for your error:"))
+    def resources(self):
+        display(Markdown("Still stuck? Here's some useful resources:"))
+        """create a submit button for the textbox"""
+        b1 = widgets.Button(description="Textbook",icon="square",
+                                               layout=widgets.Layout(width='20%', min_width='80px'))
+        b2 = widgets.Button(description="Data 8 Reference", icon="square",
+                                               layout=widgets.Layout(width='30%', min_width='80px'))
+        b3 = widgets.Button(description="Office Hours",icon="square",
+                                               layout=widgets.Layout(width='20%', min_width='80px'))
+        output = widgets.Output()
+        """aligns buttons horizontally"""
+        h1 = widgets.HBox(children=[b1,b2,b3])
+        display(h1)
+
+        def b1_click(b):
```

Comment:
What's the purpose of having three separate functions? How about one function that takes in a URL as an argument and the button reference, like
```
def button_click(button_ref, url):
  with output: 
    webbrowser.open(url)
```